subprojects/docs/src/docs/userguide/dependencyCache.adoc

```diff
@@ -0,0 +1,47 @@
+[[dependency_cache]]
+== The Dependency Cache
```

Comment:
Would you say an accurate summary of the content here to be "How Gradle's Dependency Cache Works"?

I have a hard time imagining what questions users would expect to have answered by this page. I know you probably didn't write this, but I feel like we could reduce a lot of this. If we want to keep it, we should plan on adding some diagrams of example directory structures to illustrate.

Reply:
Happy to change the title. That's probably a good idea. In a way I feel like "Separate metadata cache", "Repository caches are independent" could also be used as marketing points on `gradle.org` to some extent and remove from the user guide. Do you have any thought about it? My initial reaction was to just delete the stuff but then on the other hand it's somewhat useful. I think it's a matter of audience. If we do remove these points then I would happily include the section into the "introduction". At the moment there's just too much information for it to be in the introduction.

> If we want to keep it, we should plan on adding some diagrams of example directory structures to illustrate.

That's a good idea. I can come up with something. The problem with a directory structure is that it will likely become out-of-date as the cache is versioned. That means directory name can be different in a later version of Gradle.

Reply:
Here's a different idea. Let us consider this a part of separate would-be-contributor documentation. 

I've been meaning to collect all our knowledge about various parts of Gradle, videos, architecture docs, etc. in one place. I think this may fit well there.

The only other audience I see for this is plugin authors. I would be happy to consider changes that shift this text in that direction.

Reply:
> In a way I feel like "Separate metadata cache", "Repository caches are independent" could also be used as marketing points on gradle.org to some extent and remove from the user guide.

We already have similar words on gradle.org/features. Perhaps a bit could be added there, but that's a different audience than those who really want to know how Gradle works, which I think is what "Dependency Cache" is about.

Reply:
Almost all of those points are implementation details from my perspective. Maybe we should just delete the points.